<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>channel_dialog</class>
 <widget class="QDialog" name="channel_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通道编辑</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_channelNum">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>控制源：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_phase">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>控制类型</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioPhase_vehicle">
      <property name="text">
       <string>机动车</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioPhase_ped">
      <property name="text">
       <string>行人</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioPhase_follow">
      <property name="text">
       <string>跟随</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioPhase_other">
      <property name="text">
       <string>其他</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_type">
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>闪灯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>常绿</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>常红</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_flash">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>闪光模式</string>
   </property>
   <widget class="QRadioButton" name="radioFlash_alternate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>交替</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioFlash_red">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>红闪</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioFlash_yel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>黄闪</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioFlash_oth">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>其他</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_dim">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>辉度模式</string>
   </property>
   <widget class="QRadioButton" name="radiodim_alternate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>交替</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiodim_red">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>红灯</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiodim_yel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>黄灯</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiodim_gre">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>绿灯</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_countdown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>连接倒计时牌</string>
   </property>
   <widget class="QLabel" name="label_countdown">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>倒计时地址：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_countdown">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>28</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>启用通讯倒计时</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_id">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>28</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>控制方向</string>
   </property>
   <widget class="QComboBox" name="comboBox_Position">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>其他</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>东</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>东北</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>东南</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西南</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西北</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_Direction">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>其他</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>左转</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>直行</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>右转</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>行人</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>掉头</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>非机动</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>方位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>方向：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
